File: seunghoon/Cheese/src/main/java/com/hknu/entity/Branch.java
package com.hknu.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@AllArgsConstructor
@RequiredArgsConstructor
public class Branch {
	private final Integer branch_id;
	private final String name;
	private final float latitude;
	private final float longitude;
	private Integer shooting_cost;
	private Integer printing_cost;
	private final int paper_amount;
}

File: junghyeok/cheese_front/entity/Branch.js
import axios from "axios";

class Branch{
    constructor(id, name, longitude, latitude, shooting_cost, printing_cost, paper_amount){
        this.id = id;
        this.name = name;
        this.longitude = longitude;
        this.latitude = latitude;
        this.shooting_cost = shooting_cost;
        this.printing_cost = printing_cost;
        this.paper_amount = paper_amount;
    }

    async getAddress(){
        try{
            const res = await axios.get('/api/getAddress',{
                params:{
                    request:"coordsToaddr",
                    coords:`${this.longitude},${this.latitude}`,
                    sourcecrs:"epsg:4326",
                    output:"json",
                    orders:"legalcode"
                }
            })

            let address = [];
            for(let r in res.data.results[0].region)
            address.push(res.data.results[0].region[r].name);
            this.address = address.splice(1,).join(' ');

        } catch(error){
            console.log(error);
            throw new Error('Failed to fetch data');
        }
    }
}

export default Branch;